guides/no-noneq-in-eq.md

```diff
@@ -0,0 +1,107 @@
+# What
+
+CLC has approved the [proposal to remove `(/=)` from `class Eq`](https://github.com/haskell/core-libraries-committee/issues/3). 
+It means that `class Eq` will contain a single member `(==)` and `(/=)` will be just a normal function available from `Prelude`.
+
+Before:
+
+```haskell
+class Eq a where
+  (==) :: a -> a -> Bool
+  (/=) :: a -> a -> Bool
+```
+
+After:
+
+```haskell
+class Eq a where
+  (==) :: a -> a -> Bool
+
+(/=) :: Eq a => a -> a -> Bool
+```
+
+# When
+
+The proposal was approved conditionally, pending a resolution of 
+[-Winline-rule-shadowing: Different for single and multi method classes?](https://gitlab.haskell.org/ghc/ghc/-/issues/20535) issue. 
+At the very earliest the change may appear in GHC 9.4 (~ Fall 2022), but given that a feature freeze for 9.4 is coming soon, 
+it's more likely to happen in GHC 9.6 (~ Summer 2023).
```

Comment:
Referencing time by things like "fall" and "summer" are northern hemisphere centric (not to mention that most English speaking countries say "autumn" instead of fall).
Can I recommend "second quarter of 2022" and "mid 2023"?

Reply:
Thanks, fixed!